scripts/run_doxygen.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+doxygen_executable=${1:-doxygen}
```

Comment:
Pondering this a bit more - Should we drop this magic "fallback to whatever doxygen is on the path" stuff, and instead just *always* require the script to have the doxygen binary specified? I'm completely open either way, just wondering what the "least surprises" route is.

Reply:
The fallback makes it easier to run locally. I don't think that it's surprising that the default is whatever `doxygen` resolves to.